.github/workflows/support-window.yml

```diff
@@ -0,0 +1,55 @@
+name: Update support window diagram
+on:
+  # When translations are added/removed
+  pull_request:
+    branches: master
+    paths: README*.md
+  # Manually, when TypeScript is released
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
```

Comment:
Runs automatically when translations are added/removed, manually when TypeScript is released.